types/entities.ts

```diff
@@ -1,3 +1,5 @@
+export type ID = number | `${number}`;
```

Comment:
Keep only the first letter uppercase for types (so `Id`).  

Part of me thinks we could use a more descriptive name. What do you think of `ArchiveId`? Or something. 


`Id` is fine too,  just very broad. It could easily get confusing if people are using their own `Id` type in their libraries and have to rename this, for example.